From fe274b8f965582fdf97e6c46f90b9e7c124b0b8b Mon Sep 17 00:00:00 2001
From: Florian Festi <ffesti@redhat.com>
Date: Fri, 16 Dec 2022 15:50:12 +0100
Subject: [PATCH] rpm2archive: Don't print usage on no arguments

given as we want to default to reading from stdin and writing to stdout in
that case.
---
 rpm2archive.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/rpm2archive.c b/rpm2archive.c
index 09da8d16b..53f047f58 100644
--- a/rpm2archive.c
+++ b/rpm2archive.c
@@ -241,10 +241,6 @@ int main(int argc, const char *argv[])
 	    exit(EXIT_FAILURE);
 	}
     }
-    if (argc < 2 || poptGetNextOpt(optCon) == 0) {
-	poptPrintUsage(optCon, stderr, 0);
-	exit(EXIT_FAILURE);
-    }
 
     rpmts ts = rpmtsCreate();
     rpmVSFlags vsflags = 0;
-- 
2.38.1

